ddtrace/tracer/span.go

```diff
@@ -942,15 +942,17 @@ func setLLMObsPropagatingTags(ctx context.Context, spanCtx *SpanContext) {
 	spanCtx.trace.setPropagatingTag(keyPropagatedLLMObsMLAPP, llmSpan.MLApp())
 }
 
-// used in internal/civisibility/integrations/manual_api_common.go using linkname
+// getMeta gets a string tag. This method is safe for concurrent use and should be the primary method
+// of getting a meta tag if the span is not already locked.
 func getMeta(s *Span, key string) (string, bool) {
 	s.mu.RLock()
 	defer s.mu.RUnlock()
 	val, ok := s.meta[key]
 	return val, ok
 }
 
-// used in internal/civisibility/integrations/manual_api_common.go using linkname
+// getMetric gets a string tag. This method is safe for concurrent use and should be the primary method
+// of getting a metric tag if the span is not already locked.
```

Comment:
I'd add both comments, so we don't forget it's being used with linkname elsewhere.